cmake/modules/FindTbb.cmake

```diff
@@ -40,10 +40,14 @@ if(NOT TBB_INCLUDE_DIRS)
   set(TBB_INCLUDE_DIRS "")
 endif()
 find_path(TBB_INCLUDE_DIRS
```

Comment:
I had a minute to experiment. Your bug is here. When TBB_INCLUDE_DIRS is already defined, the find_path seems to fail (or not run). Change the find_path target variable to TBB_INCLUDE, test for TBB_INCLUDE in the downstream places and it appears to work fine.

Reply:
Thanks for that!